Cargo.toml

```diff
@@ -55,3 +55,5 @@ serde = ["dep:serde", "curve25519-dalek/serde"]
 alloc = ["curve25519-dalek/alloc", "serde?/alloc", "zeroize?/alloc"]
 precomputed-tables = ["curve25519-dalek/precomputed-tables"]
 reusable_secrets = []
+# docs: docsrs and doctest features
+docsrs = ["rand_core/getrandom"]
```

Comment:
curve25519-dalek seems to solve this by enabling the `getrandom` feature under `dev-dependencies`.

As far as an actual feature goes, I'd suggest calling this one `getrandom`

Reply:
Oh wait yeah doctests ran with `dev-dependencies` - oh yeah it was just bench that needed it oops!
Have replaced with `dev-dependencies` to align with curve :+1:

Reply:
Also there is getrandom feature now as well which provides the convenience functions which have their own tests
- https://github.com/dalek-cryptography/x25519-dalek/pull/113